docs/Routes.md

```diff
@@ -34,8 +34,7 @@ fastify.route(options)
 They need to be in
   [JSON Schema](https://json-schema.org/) format, check [here](Validation-and-Serialization.md) for more info.
 
-  * `body`: validates the body of the request if it is a POST or a
-    PUT.
+  * `body`: validates the body of the request if it is a POST, PUT or PATCH method, and doesn't work with GET and HEAD methods.
```

Comment:
Not sure about the formulation. @jsumners as english native could you rephrase this?

Reply:
maybe @Fdawgs can helps too.